descope/auth.py

```diff
@@ -222,6 +222,22 @@ def refresh_token(self, refresh_token: str) -> dict:
         resp = response.json()
         return self._generate_auth_info(resp, refresh_token)
 
+    def exchange_access_key(self, access_key: str) -> dict:
+        uri = Auth._compose_exchange_access_key_url()
+        server_response = self.do_get(uri, None, None, access_key)
+
+        json = server_response.json()
+        response = {
+            "keyId": json.get("keyId", ""),
+            "exp": json.get("expiration", 0),
+        }
+
+        jwt = json.get("sessionJwt", "")
+        if jwt:
+            response[SESSION_TOKEN_NAME] = self._validate_token(jwt)
```

Comment:
I think it worth raise exception if sessionJwt not exist as anyway the level above will have nothing to do with this.. wdyt?